packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/Content.tsx

```diff
@@ -63,111 +51,120 @@ const UserAccountMenu = ({
+            data-navi-id="profile"
           >
-          </Trans>
-        </p>
-        {user.email !== user.nichandle && (
-          <p className="mb-0">
-            <Trans
-              t={t}
-              i18nKey="user_account_menu_user_id"
-              values={{ nichandle: user.nichandle }}
-            ></Trans>
+            {t('user_account_menu_profile')}
+          </a>
+          <p className="mb-0" data-navi-id="account-email">
+            {user.email}
           </p>
-        )}
-
-        <p className={`ml-0 oui-badge oui-badge_warning`} data-navi-id="account-auth-method">
-          <strong>{t(`user_account_menu_role_${user.auth.method}`)}</strong>
-        </p>
-
-        {!user.enterprise && (
-          <UserDefaultPaymentMethod
-            defaultPaymentMethod={defaultPaymentMethod}
-            isLoading={isLoading}
-          />
-        )}
-        <hr />
-        <a
-          href={feedbackUrl}
-          className={`${style.feedback} d-flex`}
-          target="_blank"
-          rel="noopener noreferrer"
-          aria-label={t('user_account_menu_beta_feedback')}
-          onClick={onGiveFeedbackLinkClick}
-        >
-          <span className="align-self-center">
-            <img src={icon} alt={t('user_account_menu_beta_feedback')} />
-          </span>
-          <span className="pl-2">
-            <span className={`${style.feedback_title} d-block oui-link_icon`}>
-              <span>{t('user_account_menu_beta_feedback')}</span>
-              <span
-                className="oui-icon oui-icon-arrow-right ml-2"
-                aria-hidden="true"
-              ></span>
-            </span>
-            <span className={`${style.feedback_text} d-block`}>
-              {t('user_account_menu_beta_feedback_text')}
-            </span>
-          </span>
-        </a>
-        <hr />
-        <a
-          onClick={onMyAccountLinkClick}
-          className="d-block oui-link_icon"
-          aria-label={t('user_account_menu_profile')}
-          title={t('user_account_menu_profile')}
-          href={myAccountLink}
-          target="_top"
-          id="user-account-menu-profile"
-          data-navi-id="profile"
-        >
-          {t('user_account_menu_profile')}
-          <span
-            className={`${style.myAccountIcon} oui-icon oui-icon-arrow-right`}
-            aria-hidden="true"
-          ></span>
-        </a>
+          {user.email !== user.nichandle && (
+            <p className="mb-0">
+              <Trans
+                t={t}
+                i18nKey="user_account_menu_user_id"
+                values={{ nichandle: user.nichandle }}
+              ></Trans>
+            </p>
+          )}
+        </div>
+        <div className="border-bottom pb-2 pt-2">
+          <div className="d-flex justify-content-between">
+            <span>{t('user_account_menu_role_connexion')}</span>
+            <p
+              className={`ml-0 oui-chip oui-badge_warning mb-0`}
```

Comment:
I think the badge type should be dynamic.
But if not, at least it should be a green badge (oui-badge_success) and not a yellow  (oui-badge_warning) one (see mockup)